config/initializers/filter_parameter_logging.rb

```diff
@@ -50,3 +50,6 @@
 
 # Assessment filters
 Rails.application.config.filter_parameters += %i[symptomatic]
+
+# History Items
+Rails.application.config.filter_parameters += %i[comment]
```

Comment:
What about all the `notes` fields? `exposure_notes`, `close_contact_notes`, `travel_related_notes`

Reply:
History does not have those fields. They are included a few lines above under the Patient comment